crates/router/src/compatibility/stripe/webhooks.rs

```diff
@@ -101,6 +110,82 @@ pub struct StripeMandateResponse {
     pub payment_method: String,
 }
 
+#[cfg(feature = "payouts")]
+#[derive(Clone, Serialize, Debug)]
+pub struct StripePayoutResponse {
+    pub id: String,
+    pub amount: i64,
```

Comment:
can we use MinorUnit instead of plane i64 value going forward for amount

Reply:
will raise a seperate pr for refactoring amount